examples/kitchensink/server.go

```diff
@@ -516,6 +517,37 @@ func (app *KitchenSink) handleText(message *webhook.TextMessageContent, replyTok
 				return app.replyText(replyToken, err.Error())
 			}
 		}
+	case "with http info":
+		resp, _, _ := app.bot.ReplyMessageWithHttpInfo(
+			&messaging_api.ReplyMessageRequest{
+				ReplyToken: replyToken,
+				Messages: []messaging_api.MessageInterface{
+					messaging_api.TextMessage{
+						Text: "Hello, world",
+					},
+				},
+			},
+		)
+		log.Printf("status code: (%v), x-line-request-id: (%v)", resp.StatusCode, resp.Header.Get("x-line-request-id"))
+	case "with http info error":
+		resp, _, err := app.bot.ReplyMessageWithHttpInfo(
+			&messaging_api.ReplyMessageRequest{
+				ReplyToken: replyToken + "invalid",
+				Messages: []messaging_api.MessageInterface{
+					messaging_api.TextMessage{
+						Text: "Hello, world",
+					},
+				},
+			},
+		)
+		if err != nil && resp.StatusCode >= 400 && resp.StatusCode < 500 {
+			decoder := json.NewDecoder(resp.Body)
+			errorResponse := &messaging_api.ErrorResponse{}
+			if err := decoder.Decode(&errorResponse); err != nil {
+				log.Fatal("failed to decode JSON: %w", err)
+			}
+			log.Printf("status code: (%v), x-line-request-id: (%v), error response: (%v)", resp.StatusCode, resp.Header.Get("x-line-request-id"), errorResponse)
```

Comment:
Is there any good idea not to decode here?
Probably we can use https://github.com/line/line-openapi/blob/81a963ccfb2f1f058b45be2027f5334d49d414e6/messaging-api.yml#L241-L246 to decode error response on error, but few API paths supports error response type...